lib/importers/protected_food_drink_name/parser.rb

```diff
@@ -0,0 +1,461 @@
+
+        [class_category_map[data_string]]
+      end
+
+      def protection_type
+        protection_type_map[data["Protection type"]]
+      end
+
+      def reason_for_protection
+        reason_for_protection_map[data["Reason for protection"]]
+      end
+
+      def country
+        return [] if data["Country of origin"].blank?
+
+        data["Country of origin"].split(", ").map do |country_string|
+          country_map[country_string]
+        end
+      end
+
+      def traditional_term_grapevine_product_category
+        return [] if data["Traditional term grapevine product category"].blank?
+
+        data["Traditional term grapevine product category"].split(", ").map do |category_string|
+          grapevine_category_map[category_string]
+        end
+      end
+
+      def traditional_term_type
+        term_type_map[data["Traditional term type"]]
+      end
+
+      def traditional_term_language
+        language_map[data["Traditional term language"]]
+      end
+
+      def date_application
+        parse_date(data["Date of application"])
+      end
+
+      def date_registration
+        parse_date(data["Date of UK registration"])
+      end
+
+      def date_registration_eu
+        parse_date(data["Date of original registration with the EU"])
+      end
+
+      def summary
+        case data["Protection type"]
+        when "Geographical indication (GI)"
+          if data["Product type"] == "Spirit drink"
+            "Protected spirit drink name"
+          elsif data["Product type"] == "Aromatised wine"
+            "Protected aromatised wine name"
+          end
+        when "Protected Geographical Indication (PGI)"
+          if data["Product type"] == "Food"
+            "Protected food name with Protected Geographical Indication (PGI)"
+          elsif data["Product type"] == "Wine"
+            "Protected wine name with Protected Geographical Indication (PGI)"
+          end
+        when "Protected Designation of Origin (PDO)"
+          if data["Product type"] == "Food"
+            "Protected food name with Protected Designation of Origin (PDO)"
+          elsif data["Product type"] == "Wine"
+            "Protected wine name with Protected Designation of Origin (PDO)"
+          end || ""
+        when "Traditional Specialities Guaranteed (TSG)"
+          "Protected food name with Traditional Speciality Guaranteed (TSG)"
+        when "Traditional Term"
+          "Traditional term for wine"
+        when "Name protected by international treaty"
+          "Name protected by international treaty"
+        end
+      end
+
+      def body
+        content = ""
+
+        # Product specification
+        if data["Product type"] != "Traditional term"
+          content += "## Product specification \n\n" \
+            "The product specification is not available on this site. " \
+            "Find out how to [get a product specification for a protected food name] " \
```

Comment:
```suggestion
            "Find out how to [get a product specification for a protected food name]" \
```

This space means the links aren't rendered correctly.